Guided Hunting - Azure Resource Explorer.ipynb

```diff
@@ -0,0 +1,7311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    display(HTML("
Starting Notebook setup...
"))
</pre><p>You should simplify this whole section to:</p><p>(I'm assuming that you don't need the extra imports of ipwhois, urlretrieve and yaml - I don't see these used anywhere.</p><pre class="ql-syntax" spellcheck="false">from IPython.display import display, HTML, Markdown

REQ_PYTHON_VER=(3, 6)
REQ_MSTICPY_VER=(0, 6, 0)

display(HTML("Checking for msticpy update"))
%pip install --upgrade msticpy
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;
import msticpy
msticpy.init_notebook(namespace=globals())
<code></code>`
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>done!</p>